### ISSUE TITLE
Get JS SourceMaps in dist when using ES6 with webpack/Babel

### ISSUE THREAD

**Issue by ulilicht** [Swiip/generator-gulp-angular#865]
Hello, 

I'm using ES6 with Babel. Is there a way to get JavaScript SourceMaps in the build task? 
If I just put ```.pipe($.sourcemaps.init())```and ```.pipe($.sourcemaps.write('maps'))``` in the build.js file, I get SourceMaps which point from ES5 Minified to the output of Babel. 

It would be great to get maps which show the original code. 

Thanks for your help!

**Comment by Swiip** [comment]
Yep, I also wanted that but I didn't find a way 'till now. The problem is that there is two separate processes: webpack and useref. Webpack is writing in .tmp and useref from .tmp to dist.

It's perhaps possible but I didn't find a way. Perhaps sourcemaps from useref is able to take the mapping from webpack as an entry point? I know I succeeded such a thing for ruby sass.
